File: api/app/Models/GlobalSettings.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;

class GlobalSettings extends Model
{
    protected $fillable = ['sender_name', 'sender_street', 'sender_zip', 'sender_city', 'sender_phone', 'sender_mail', 'sender_vat', 'sender_bank', 'sender_web', 'service_order_comment'];
}

File: api/spec/factories/global_settings.rb
# frozen_string_literal: true

FactoryBot.define do
  factory :global_setting do
    sender_name { 'MyString' }
    sender_street { 'MyString' }
    sender_zip { '' }
    sender_phone { 'MyString' }
    sender_city { 'MyString' }
    sender_mail { 'MyString' }
    sender_vat { 'MyString' }
    sender_bank { 'MyString' }
    sender_web { 'MyString' }
    service_order_comment { 'MyString' }
    sender_bank_detail { 'MyString' }
    sender_bank_iban { 'MyString' }
    sender_bank_bic { 'MyString' }
  end
end
